src/lib/common/sem.cpp

```diff
@@ -459,13 +459,131 @@ int timeStatSemGive(const char* who, const char* what)
 
 
 /* ****************************************************************************
+*
+* reqSemGet - 
+*/
+const char* reqSemGet(void)
+{
+  int value;
+
+  if (sem_getvalue(&reqSem, &value) == -1)
+  {
+    return "error - state not available";
+  }
+
+  if (value == 0)
+  {
+    return "taken";
+  }
+  
+  return "free";  
+}
+
+
+
+/* ****************************************************************************
+*
+* transSemGet - 
+*/
+const char* transSemGet(void)
+{
+  int value;
+
+  if (sem_getvalue(&transSem, &value) == -1)
+  {
+    return "error - state not available";
+  }
+
+  if (value == 0)
+  {
+    return "taken";
+  }
+  
+  return "free";  
+}
+
+
+
+/* ****************************************************************************
+*
+* cacheSemGet - 
+*/
+const char* cacheSemGet(void)
+{
+  int value;
+
+  if (sem_getvalue(&cacheSem, &value) == -1)
+  {
+    return "error - state not available";
+  }
+
+  if (value == 0)
+  {
+    return "taken";
+  }
+  
+  return "free";  
+}
+
+
+
+/* ****************************************************************************
+*
+* timeStatSemGet - 
+*/
+const char* timeStatSemGet(void)
+{
+  int value;
+
+  if (sem_getvalue(&timeStatSem, &value) == -1)
+  {
+    return "error - state not available";
+  }
+
+  if (value == 0)
+  {
+    return "taken";
+  }
+  
+  return "free";  
+}
+
+
+
+/* ****************************************************************************
 *  curl context
 */
 
+static bool contexts_mutex_taken   = false;
```

Comment:
I uderstand that according to M6 style rule (https://github.com/telefonicaid/fiware-orion/blob/develop/doc/manuals/contribution_guidelines.md#m6-naming-conventions) this names should be camelcase. 

(I'm not in favour of changing names of existing variables except in "pure" style change PRs but at least let's use right names for new variables).